recipe/meta.yaml

```diff
@@ -186,6 +188,11 @@ outputs:
         - osgeo._gdalconst
         - osgeo._ogr
         - osgeo._osr
+      commands:
+        - $PYTHON run_tests.py  # [unix]
+        - %PYTHON% run_tests.py  # [win]
+        - $PYTHON extra_tests.py  # [unix]
+        - %PYTHON% extra_tests.py  # [win]
       about:
```

Comment:
... but I still can't work out how to get these tests to run. Any ideas?
```
can't open file 'run_tests.py': [Errno 2] No such file or directory
```

Reply:
@gillins It's `run_test.py` not `run_tests.py`

Reply:
:man_facepalming: well spotted!

Reply:
@gillins Looking at `run_test.py`, `extra_tests.py` is imported there. So we do not need to call it specifically, or do I miss something?

Reply:
Not yet, still thinking...